s3/variables.tf

```diff
@@ -77,3 +77,23 @@ variable "force_destroy" {
   description = "destroys S3 bucket on terraform destroy, even if there are files inside the bucket"
   default     = true
 }
+
+variable "sns_topic_arn" {
```

Comment:
Should these variable name be more specific to log aggregation? Just thinking if we need to add additional sns topics in the future to S3 buckets

Reply:
Thanks, separated SNS variables into log SNS variables, for log bucket, and the SNS variables for main S3 bucket